src/main.cpp

```diff
@@ -10,16 +10,15 @@ All rights reserved (see LICENSE).
 #include <fstream>
 #include <iostream>
 #include <sstream>
-#ifndef _WIN32
-#include <unistd.h>
-#else
-#include "getopt_win.h"
-#endif
 
 #if USE_LIBOSRM
 #include "osrm/exception.hpp"
 #endif
 
+// so we don't split heuristic param by comma
+#define CXXOPTS_VECTOR_DELIMITER ';'
```

Comment:
keep this or change `heuristic_params_arg` separator? or maybe another value instead of `;`?

Reply:
Thanks for the explanation. Let's keep the default behavior (so remove this definition) and simply change our own internal separator:

```patch
diff --git a/src/main.cpp b/src/main.cpp
index bc9c6bd..f9ab8c7 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -15,8 +15,6 @@ All rights reserved (see LICENSE).
 #include "osrm/exception.hpp"
 #endif
 
-// so we don't split heuristic param by comma
-#define CXXOPTS_VECTOR_DELIMITER ';'
 #include "../include/cxxopts/include/cxxopts.hpp"
 
 #include "problems/vrp.h"
diff --git a/src/utils/helpers.h b/src/utils/helpers.h
index a2c1c98..3e64e11 100644
--- a/src/utils/helpers.h
+++ b/src/utils/helpers.h
@@ -108,7 +108,7 @@ inline void log_LS_operators(
 
 inline HeuristicParameters str_to_heuristic_param(const std::string& s) {
   // Split command-line string describing parameters.
-  constexpr char delimiter = ',';
+  constexpr char delimiter = ';';
   std::vector<std::string> tokens;
   std::string token;
   std::istringstream tokenStream(s);
```

 This means we should now use `-e "0;NONE;0.3"` instead of `-e "0,NONE,0.3"`, but as this is an (undocumented) debug option we're not breaking anything.